home/content_import_export.py

```diff
@@ -238,6 +239,7 @@ def add_whatsapp(row, whatsapp_messages, page=None, variation_messages=()):
             return page
 
         whatsapp_template_name = row.get("whatsapp_template_name", "")
+        whatsapp_template_category = row.get("whatsapp_template_category") or "UTILITY"
```

Comment:
Because there could be a `whatsapp_template_category` column, but it has no value for that row. In that case if we do `row.get("whatsapp_template_category", "UTILITY")`, we'll get `""` returned, which is not a value value for this field. We want the default here to be `UTILITY`, so if they leave it blank we should choose a correct default.